diff --git a/node_modules/@oauth-everything/passport-twitch/dist/Strategy.js b/node_modules/@oauth-everything/passport-twitch/dist/Strategy.js
index 69ef910..ffd9a7a 100644
--- a/node_modules/@oauth-everything/passport-twitch/dist/Strategy.js
+++ b/node_modules/@oauth-everything/passport-twitch/dist/Strategy.js
@@ -10,10 +10,18 @@ class Strategy extends passport_oauth2_1.Strategy {
             ...options
         }, verify);
         this.name = "twitch";
+        this.clientID = options.clientID;
     }
     userProfile(accessToken, done) {
         this._oauth2.useAuthorizationHeaderforGET(true);
-        this._oauth2.get("https://api.twitch.tv/helix/users", accessToken, (error, result) => {
+
+        let headers = {
+            'Authorization': "Bearer " + accessToken,
+            'Client-ID': this.clientID
+        };
+
+        this._oauth2._request("GET", "https://api.twitch.tv/helix/users", headers, "", "", (error, result) => {
+
             if (error)
                 return done(new passport_oauth2_1.InternalOAuthError("Failed to fetch user profile", error));
             let json;
