--- schema.hasura.user.graphqls
+++ schema.nhost.user.graphqls
@@ -1467,7 +1467,7 @@ type mutation_root {
   league: leagueMutation
   recordMyTeamGame(isHome: Boolean!, opponentId: ID!, opponentScore: Int!, teamScore: Int!): Game!
   reportGame(awayScore: Int!, awayTeamId: ID!, homeScore: Int!): Game!
-  set_department_manager(args: set_department_manager_args! distinct_on: [user_departments_select_column!] limit: Int offset: Int order_by: [user_departments_order_by!] where: user_departments_bool_exp): user_departments
+  set_department_manager(args: set_department_manager_args!): user_departments
   updateFile(_append: files_append_input _delete_at_path: files_delete_at_path_input _delete_elem: files_delete_elem_input _delete_key: files_delete_key_input _inc: files_inc_input _prepend: files_prepend_input _set: files_set_input pk_columns: files_pk_columns_input!): files
   updateFiles(_append: files_append_input _delete_at_path: files_delete_at_path_input _delete_elem: files_delete_elem_input _delete_key: files_delete_key_input _inc: files_inc_input _prepend: files_prepend_input _set: files_set_input where: files_bool_exp!): files_mutation_response
   updateUser(_append: users_append_input _delete_at_path: users_delete_at_path_input _delete_elem: users_delete_elem_input _delete_key: users_delete_key_input _prepend: users_prepend_input _set: users_set_input pk_columns: users_pk_columns_input!): users
@@ -1882,8 +1882,7 @@ type query_root {
   files(distinct_on: [files_select_column!] limit: Int offset: Int order_by: [files_order_by!] where: files_bool_exp): [files!]!
   game(id: ID!): Game
   games: [Game!]!
-  get_department_manager(args: get_department_manager_args! distinct_on: [user_departments_select_column!] limit: Int offset: Int order_by: [user_departments_order_by!] where: user_departments_bool_exp): user_departments
-  get_department_manager_aggregate(args: get_department_manager_args! distinct_on: [user_departments_select_column!] limit: Int offset: Int order_by: [user_departments_order_by!] where: user_departments_bool_exp): user_departments_aggregate!
+  get_department_manager(args: get_department_manager_args!): user_departments
   identity_check_logs(distinct_on: [identity_check_logs_select_column!] limit: Int offset: Int order_by: [identity_check_logs_order_by!] where: identity_check_logs_bool_exp): [identity_check_logs!]!
   identity_check_logs_by_pk(id: bigint!): identity_check_logs
   kb_entries(distinct_on: [kb_entries_select_column!] limit: Int offset: Int order_by: [kb_entries_order_by!] where: kb_entries_bool_exp): [kb_entries!]!
@@ -1977,8 +1976,7 @@ type subscription_root {
   file(id: uuid!): files
   files(distinct_on: [files_select_column!] limit: Int offset: Int order_by: [files_order_by!] where: files_bool_exp): [files!]!
   files_stream(batch_size: Int! cursor: [files_stream_cursor_input]! where: files_bool_exp): [files!]!
-  get_department_manager(args: get_department_manager_args! distinct_on: [user_departments_select_column!] limit: Int offset: Int order_by: [user_departments_order_by!] where: user_departments_bool_exp): user_departments
-  get_department_manager_aggregate(args: get_department_manager_args! distinct_on: [user_departments_select_column!] limit: Int offset: Int order_by: [user_departments_order_by!] where: user_departments_bool_exp): user_departments_aggregate!
+  get_department_manager(args: get_department_manager_args!): user_departments
   identity_check_logs(distinct_on: [identity_check_logs_select_column!] limit: Int offset: Int order_by: [identity_check_logs_order_by!] where: identity_check_logs_bool_exp): [identity_check_logs!]!
   identity_check_logs_by_pk(id: bigint!): identity_check_logs
   identity_check_logs_stream(batch_size: Int! cursor: [identity_check_logs_stream_cursor_input]! where: identity_check_logs_bool_exp): [identity_check_logs!]!
